release.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
@twolfson @dawsonbotsford

One is guarantied that `/bin/sh` is on all systems. And since the code is POSIX compatible - and the file is named release.*sh* - then there is no need to load `bash` as a faster / more lightweight shell like `dash` could be symlinked from `/bin/sh`. [Also note that FreeBSD doesn't ship with `bash` by default](https://www.freebsd.org/doc/en/articles/linux-users/shells.html)

The discussion linked above is concerned about whether or not to use `/usr/bin/env sh` or simply `/bin/sh` which makes sense when talking about `bash` as a user wish to use a newer version of said. But whom are not allowed to change `/bin`.

But since - as stated before - this code is POSIX compatible the discussion is not as relevant. One could however still argue that they have symlinked `/usr/bin/env sh` to `dash` but the system's version is `bash` in which case the performance boost will be lost. But using `/usr/bin/env` one extra process needs to be started which also cost something.

Anyway, the main idea behind changing the shebang was that a `.sh` file should be POSIX compatible and if a shebang is present it should be for the `sh` executable. I think you can compare it to saving a png file with the `.jpg` suffix. One could also rename the file to `release.bash` or `release`.